### DISCUSSION TITLE
Dịch scheduler thành hoạch định

### DISCUSSION THREAD

**Discussion by ngcthuong** [mlbvn/handson-ml2-vn#119] [Terminology] [answered]
HIện tại schedule/scheduling đang được dịch là định thời. Mình thấy định thời phù hợp với nghĩa timer hơn, cụ thể mình học ĐTVT vẫn dịch bộ timer là bộ định thời. 

Với từ scheduling mình thấy hợp với nghĩa "hoạch định" hơn. 

Mọi người thấy thế nào?

cc @mlbvn/handson-ml

**Comment by minhduc0711** [comment, marked as answer]
Từ `định thời` trước em tham khảo bên môn hệ điều hành ([ref](http://www.cse.hcmut.edu.vn/~hungnq/courses/lopchuyendoi/ch05-CPU-Scheduling.pdf)), nhưng em thấy từ `hoạch định` cũng ổn ạ.

EDIT: chốt đổi thành `hoạch định`

**Reply by ngcthuong** [reply]
Mình thấy từ "thời" nó hay ý liên quan tới thời gian, nhưng điểu chỉnh learning rate hoàn toàn ko liên quan tới thời gian mà liên quan tới số lần chạy epoch, số lần duyệt qua dữ liệu hơn.

**Reply by hnguyentt** [reply]
@minhduc0711 thống nhất rồi thì Đức update trên glossary luôn đi. Trên đó vẫn đang là định thời.
